storage/src/store/block_handle/handle.rs

```diff
@@ -72,6 +72,10 @@ impl BlockHandle {
         self.inner.meta.flags().contains(BlockFlags::IS_APPLIED)
     }
 
+    pub fn is_persistent(&self) -> bool {
+        self.inner.meta.flags().contains(BlockFlags::IS_PERSISTENT) || self.inner.id.seqno == 0
```

Comment:
Where can we get blocks with 0 seqno?

Reply:
Block handles are created even for zerostates